src/package.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
My only concern is this standalone tag inserted by @force2b and removed by @randi274.

Any idea if it could be a problem, @cpolcyn @jlantz?

Reply:
Ran into this the other day in another project and I think it had no effect really. I think if build passes we let this slide.